docs/manual/guide/production/ConductR.md

```diff
@@ -26,63 +48,92 @@ The above creates what is known as a "bundle". A bundle is the unit of deploymen
+In the previous [Installing ConductR sandbox](#Installing-ConductR-sandbox) section we have already installed the `conductr-cli`. The CLI can be used on the terminal to communicate with the ConductR cluster. 
 
-To access ConductR's sandbox from within the activator console first added the following sbt plugin to your `project/plugins.sbt`:
+To use the same commands within the activator console add the following sbt plugin to your `project/plugins.sbt`:
 
 ```scala
 addSbtPlugin("com.typesafe.conductr" % "sbt-conductr-sandbox" % "1.4.2")
 ```
 
-Additionally it is necessary to install the [conductr-cli](https://github.com/typesafehub/conductr-cli). The CLI is used by the sandbox to communicate with the ConductR cluster. Follow the "Getting Started" section on the [ConductR Developer](https://www.lightbend.com/product/conductr/developer) page to install the conductr-cli.
-
-To start the sandbox from within the console you also need to specify the ConductR version. The latest version can be picked up from the [ConductR Developer](https://www.lightbend.com/product/conductr/developer) page as well:
+To start the sandbox from within the console you also need to specify the ConductR version. Please visit the [ConductR Developer page](https://www.lightbend.com/product/conductr/developer) again to pick up the latest ConductR version from the section **Quick Configuration**:
 
 ```console
 > reload
-> set SandboxKeys.imageVersion in Global := "YOUR_CONDUCTR_SANDBOX_VERSION"
-> sandbox run
+> set SandboxKeys.imageVersion in Global := "YOUR_CONDUCTR_VERSION"
+> sandbox run --with-features logging
 ```
 
-With the sandbox running you can now load the bundle that you previously generated:
+The `reload` command loads the previously added `sbt-conductr-sandbox` plugin to your project. The `SandboxKeys.imageVersion` key is used by the `sandbox run` command to start a specific version of the the ConductR cluster locally. In this case we are starting the sandbox with the logging feature in order to retrieve logging information of the bundles.
+
+With the ConductR sandbox running you can now load the bundle that you previously generated:
 
 ```console
-> my-service-impl/conduct load <press the tab key here>
+> project my-service-impl
+> conduct load <press the tab key here>
 ```
 
-Note how you're loading the implementation of your service.
+When starting the activator console you are in the context of the `root` project. However, the bundles are created for your sub projects, i.e. your service implementations. Therefore, it is necessary to first switch with `project my-service-impl` to the sub project of the service you want to load on to ConductR. Replace `my-service-impl` with the name of your sub project.
```

Comment:
I'd remove the backticks from root, since by default the root project is named after the folder that contains it.